apis/config/v1beta1/groupversion_info.go

```diff
@@ -14,10 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha2 contains API Schema definitions for the config v1alpha2 API group
+// Package v1beta1 contains API Schema definitions for the config v1beta1 API group
 // +kubebuilder:object:generate=true
 // +groupName=config.x-k8s.io
```

Comment:
@alculquicondor Which group name is corrected? `config.x-k8s.io` vs `config.kueue.x-k8s.io`?

Reply:
`config.kueue.x-k8s.io`

Reply:
Thanks. I updated this line.